openai/openai.go

```diff
@@ -86,6 +92,38 @@ type ChatCompletionChunk struct {
 	Choices           []ChunkChoice `json:"choices"`
 }
 
+type CompletionRequest struct {
+	Model            string   `json:"model"`
+	Prompt           string   `json:"prompt"` // openai also accepts a list of strings if we want to do that
+	FrequencyPenalty *float64 `json:"frequency_penalty"`
+	MaxTokens        *int     `json:"max_tokens"`
```

Comment:
I don't think we need pointers for any of these

Reply:
This struct was modeled after the one used for chat, which had pointers for the same fields. If we wanna change some of these we'd change those too, let me know